[IMP] Localizations/Malaysia: improve doc (#13639)
odoo/documentation: content/applications/finance/fiscal_localizations/malaysia.rst

   * - :guilabel:`Malaysia - E-invoicing`
     - `l10n_my_edi`
     - This module includes the features required for integration with MyInvois under IRBM.

>    * - :guilabel:`Debit Notes`
>      - `account_debit_note`
>      - A debit note is used as an increase of the amounts of an existing invoice or in some
>        specific cases to cancel a credit note.

Reviewer: The module has to be deleted here, too.
```suggestion
```

- :guilabel:`MyInvois Credit Note`: This is created when an Odoo credit note is reconciled with the
  original invoice.
- :guilabel:`MyInvois Refund Note`: This is created when an Odoo credit note is reconciled with a
  full payment instead of the original invoice.

.. note::
   If a credit note is reconciled with only a partial payment before being sent, it is still
   categorized as a :guilabel:`credit note` in the MyInvois portal.

.. tip::
   To issue both a credit note and a refund note for the same original invoice:
    - Create two separate credit notes in Odoo from the original invoice.
    - For one, register a payment before sending it. This becomes a :guilabel:`Refund Note` in the
      MyInvois Portal.
    - For the other, do not register a payment before sending it. This becomes a
      :guilabel:`Credit Note`.

.. note::
   The same logic applies to credit notes created from bills: if reconciled with a full payment, it
   becomes a :guilabel:`Self-billed Refund Note`; Otherwise, it becomes a
   :guilabel:`Self-billed Credit Note`.

Send debit note to MyInvois
~~~~~~~~~~~~~~~~~~~~~~~~~~~

> To issue a debit note from an existing bill or invoice:
> 
> #. Click the :icon:`fa-cog` :guilabel:`(gear)` icon and select :guilabel:`Debit Note`.
> 
>    .. image:: malaysia/myinvois-debit-note.png
>       :alt: MyInvois debit note
> 
> #. Fill in the necessary fields and click :guilabel:`Create Debit Note`.
> #. Complete and :guilabel:`Confirm`` the new debit note.
> #. Click :guilabel:`Send To MyInvois`. In the MyInvois Portal, it appears as either a
>    :guilabel:`Debit Note` or a :guilabel:`Self-billed Debit Note`.

Reviewer: I would remove these explanations and the image, and add a reference to the section on how to issue a debit note to avoid duplicate information. I also added some precision at the end, can you check I'm not wrong (whether it's from an invoice or a vendor bill)? ;) Thank you!
```suggestion
:ref:`Issue a debit note from an existing bill or invoice <accounting/credit_notes/issue-debit-note>` and click :guilabel:`Send To MyInvois`. In MyInvois, it appears then as a :guilabel:`Debit Note` if issued from an invoice or a :guilabel:`Self-billed Debit Note` from a vendor bill.
```

Author: Your understanding is correct. I'll just follow your revision. Thank you!
